docs/core/extensions/snippets/channels/channels/LoggingStopwatch.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Runtime.CompilerServices;
```

Comment:
Where are the rest of the code files used?

Reply:
What do you mean?

Reply:
Sorry, where are this file and the ones other than Coordinates.cs used?

Reply:
Ah, ok. There is a _Program.cs_ file, this is a top-level program... It's a `partial`, and all of the other `partial` implementations are shared with the `Program`. See: [docs/core/extensions/snippets/channels/Program.cs](https://github.com/dotnet/docs/blob/243819cd8b25968a50691329cfbeda1767377b75/docs/core/extensions/snippets/channels/Program.cs)

Reply:
Moved it up